person.rb

```diff
@@ -2,6 +2,9 @@
 require_relative 'rental'
 
 class Person
+  attr_accessor :name, :age, :rentals
+  attr_reader :id
```

Comment:
- [ ] We don't need any Getter and Setter for the `id`. Since `id` is just a variable.

Reply:
OK, thanks for the notes!